lib/manager/components/LabelEditor.js

```diff
@@ -72,24 +72,22 @@ class LabelEditor extends React.Component<Props, State> {
     }
 
   _onFormChange = ({target}: {target: HTMLInputElement}) => {
-    const {name, value} = target
+    const {checked, name, value: targetValue, type} = target
 
-    this.setState(
-      update(this.state, {
-        newLabel: { $merge: {[name]: value} },
-        validation: { [name]: { $set: value && value.length > 0 } }
-      })
-    )
-  }
+    let valid = targetValue && targetValue.length > 0
```

Comment:
This is 11th hour, but something I realized was that we should probably limit the label length to something like 25 characters. Probably OK just to handle this with `<input maxlength="25" ... />`.

Reply:
Handled in #700.